apps/learner/src/routes/content/[id]/quiz/+page.svelte

```diff
@@ -195,3 +198,44 @@
     </div>
   </div>
 {/if}
+
+{#if isCompletionModalOpen}
+  <div class="z-100 fixed inset-0 h-full w-full bg-slate-950 text-white">
+    <Starfield />
+
+    <div class="mx-auto h-full w-full max-w-5xl px-4 py-3">
+      <div class="flex h-full flex-col">
+        <!-- TODO: placeholder image, to be replaced once confirmed -->
+        <picture class="flex flex-1 justify-center">
+          <source media="(min-width: 1024px)" srcset={`${base}/meteor/384w.webp`} />
+
```

Comment:
I don't think we need newlines for `picture` element since all its children work together as a single unit. Unlike containers that include multiple independent elements like text or other blocks.